src/Castle.Windsor/MicroKernel/SubSystems/Configuration/DefaultConfigurationStore.cs

```diff
@@ -181,7 +181,8 @@ public IConfiguration[] GetInstallers()
 
 		public IResource GetResource(String resourceUri, IResource resource)
 		{
-			if (resourceUri.IndexOf(Uri.SchemeDelimiter) == -1)
+			var uriSchemeDelimiter = "://";
```

Comment:
Uri.SchemeDelimiter comes back in 2.0. @jonorossi  at you ok with adding something like  `FEATURE_URIMETHODS` for this?